vendor/github.com/openshift/api/security/v1/0000_03_security-openshift_01_scc.crd.yaml

```diff
@@ -0,0 +1,353 @@
+apiVersion: apiextensions.k8s.io/v1beta1
```

Comment:
Depending if we need to create this in code vs a static file in the manifest, we may not need any of this vendoring